{
    "checksum": "4cb332e4e6cc3823b8297c04276787e4db0ae9eac46330860226f74f7d2787c6",
    "file": "exceptions.py",
    "mime": {
        "encoding": "us-ascii",
        "type": "text/x-python"
    },
    "number_of_duplicates": 6,
    "package": "python-os-ken",
    "path": "python-os-ken/2.1.0-2/doc/source/quantumclient/common/exceptions.py",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/python-os-ken/2.1.0-2/",
        "metric": {
            "size": 21232
        },
        "pts_link": "https://tracker.debian.org/pkg/python-os-ken",
        "sloc": [
            [
                "python",
                100710
            ],
            [
                "erlang",
                14517
            ],
            [
                "ansic",
                594
            ],
            [
                "sh",
                486
            ],
            [
                "makefile",
                133
            ]
        ],
        "suites": [
            "bookworm",
            "sid"
        ],
        "vcs_browser": "https://salsa.debian.org/openstack-team/libs/python-os-ken",
        "vcs_type": "git"
    },
    "raw_url": "/data/main/p/python-os-ken/2.1.0-2/doc/source/quantumclient/common/exceptions.py",
    "stat": {
        "perms": "rw-r--r--",
        "size": 63,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": true,
    "type": "file",
    "version": "2.1.0-2"
}